common/spdat.h

```diff
@@ -1505,6 +1505,7 @@ bool IsDisciplineBuff(uint16 spell_id);
 bool IsDiscipline(uint16 spell_id);
 bool IsCombatSkill(uint16 spell_id);
 bool IsResurrectionEffects(uint16 spell_id);
+int8 DoResSicknessChecks(uint16 spell_id1, uint16 spell_id2);
```

Comment:
This is a bit oddly named. Maybe `GetResurrectionSicknessCheck`? Verbosity doesn’t hurt when naming methods.